draft-js-video-plugin/src/video/components/DefaultVideoComponent.js

```diff
@@ -0,0 +1,57 @@
+import React, { PropTypes } from 'react';
+import utils from '../utils';
+import styles from './DefaultVideoComponent.css';
+
+const YOUTUBE_PREFIX = 'https://www.youtube.com/embed/';
+const VIMEO_PREFIX = 'https://player.vimeo.com/video/';
+const iframeStyle = {
+  width: '100%',
+  height: '100%',
+  position: 'absolute',
+  top: '0',
+  left: '0',
+};
+
+const getSrc = ({ src }) => {
+  const {
+    isYoutube,
+    getYoutubeSrc,
+    isVimeo,
+    getVimeoSrc,
+  } = utils;
+  if (isYoutube(src)) {
+    const { srcID } = getYoutubeSrc(src);
+    return `${YOUTUBE_PREFIX}${srcID}`;
+  }
+  if (isVimeo(src)) {
+    const { srcID } = getVimeoSrc(src);
+    return `${VIMEO_PREFIX}${srcID}`;
+  }
+  return undefined;
+};
+
+const DefaultVideoCompoent = (props) => {
+  const { blockProps, style } = props;
+  const src = getSrc(blockProps);
+  if (src) {
+    return (
+      <div style={style} >
+        <div className={styles.iframeContainer} >
```

Comment:
sorry, I wasn't precise enough. Can we have just the iFrame? Are the divs necessary to properly render?